app/scripts/controllers/swaps.js

```diff
@@ -413,6 +451,13 @@ export default class SwapsController {
     })
   }
 
+  setSwapsQuoteRefreshTime(swapsQuoteRefreshTime) {
+    const { swapsState } = this.store.getState()
```

Comment:
Perhaps this could be inlined in `_setSwapsQuoteRefreshTime` instead? It is no longer being called externally, so it doesn't need to be a public method anymore.